common/src/main/resources/database/mysql_schema.sql

```diff
@@ -14,6 +14,7 @@ CREATE TABLE IF NOT EXISTS `%user_data_table%`
     `player_uuid`  char(36)    NOT NULL,
     `timestamp`    datetime    NOT NULL,
     `save_cause`   varchar(32) NOT NULL,
+    `server_id`    varchar(32) NOT NULL,
```

Comment:
I think just `server` would be fine here to avoid ambiguity with actual numerical/unique ID fields